Automattic/jetpack
# JS Components: Convert QRCode component to TypeScript

#### Changes proposed in this Pull Request:
* This PR converts `QRCode` component to TypeScript and cleans up the propTypes.
* It also converts the `QRCode` stories to TS

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* Checkout the PR
* Run `cd projects/js-packages/storybook && pnpm run storybook:dev`
* Confirm that all the stories of `QRCode` component work as expected
* Check the consumers of `QRCode` and related components and confirm that they work as expected
* Check for any regression

<img width="1028" alt="Screenshot 2022-05-19 at 10 55 04 AM" src="https://user-images.githubusercontent.com/18226415/169217014-c41db095-76ee-441a-9198-e41c1e37e83d.png">


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202276314264003
